composer.lock

```diff
@@ -12570,5 +12570,5 @@
     "platform-overrides": {
         "php": "7.2.5"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.3.0"
```

Comment:
I think this shouldn't have been changed :/

Reply:
It's a value automatically updated by composer, I think it depends on which version you have locally while running composer so ultimately it depends on who runs it

I don't know how we can prevent this except by "forcing" all contributors to use the same composer version